### PR TITLE
Add `DfTx` for Create & Resign MasterNode

### PR DESCRIPTION
**Pull request by ivan-zynesis**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- for explorer to decode
- to create/resign masternode in future

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * CreateMasterNode DeFi Transaction
+ */
+export interface CreateMasterNode {
+  type: number // ----------------------------------| 1 byte, if 0x01 p2pkh, otherwise = p2wpkh
+  collateralPubKeyHash: string // ------------------| VarUInt{20 bytes}
```
Comment (fuxingloh) [inline packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts:11]: missing 1 operator

**Review by fuxingloh — CHANGES_REQUESTED** [review]

File: packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Composable ResignMasterNode, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CResignMasterNode extends ComposableBuffer<ResignMasterNode> {
+  static OP_CODE = 0x52 // 'R'
+  static OP_NAME = 'OP_DEFI_TX_RESIGN_MASTER_NODE'
+
+  composers (cmn: ResignMasterNode): BufferComposer[] {
+    return [
+      ComposableBuffer.hex(32, () => cmn.nodeId, v => cmn.nodeId = v)
```
Comment (fuxingloh) [inline packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts:47]: I am assuming this is a `txid`, you might want to use `hexBEBufferLE` instead as it's the defacto way to present `txid`.

```suggestion
      ComposableBuffer.hexBEBufferLE(32, () => cmn.nodeId, v => cmn.nodeId = v)
```

File: packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts (hunk 1 above)
Comment (ivan-zynesis) [reply packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts:11]: Do not understand, can you elaborate?

Comment (canonbrother) [reply packages/jellyfish-transaction/src/script/defi/dftx_masternode.ts:11]: @ivan-zynesis here
the `operator` is needed as well
https://github.com/DeFiCh/ain/blob/e3db4110f2bcb41b4ae2d49e3fb669d11ce5cca8/src/masternodes/rpc_masternodes.cpp#L111-L113

**Comment by fuxingloh** [comment]
Failing test, wrong error message

**Review by fuxingloh — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

**Pull request merged**